src/ansible_compat/runtime.py

```diff
@@ -116,14 +117,19 @@ def clean(self) -> None:
             shutil.rmtree(self.cache_dir, ignore_errors=True)
 
     def exec(
-        self, args: Union[str, List[str]], retry: bool = False
+        self, args: Union[str, List[str]], retry: bool = False, tee: bool = False
```

Comment:
I'm not sure how I feel about seeing a million args added to this method. It seems to be exhaustive.

Reply:
I doubt we will too many args here. In fact we already have less than original run, as the universal_newlines are implicit.